assets/app/scripts/filters/resources.js

```diff
@@ -1,6 +1,16 @@
 'use strict';
 
 angular.module('openshiftConsole')
+  .filter('uid', function() {
```

Comment:
comment about what this is used for (otherwise returning the resource in the `else` case doesn't make sense)